lib/validators/stale.js

```diff
@@ -0,0 +1,52 @@
+const { Validator } = require('./validator')
+
+class Stale extends Validator {
+  constructor () {
+    super()
+    this.supportedEvents = [
```

Comment:
The event here is wrong. It should be `schedule` -- and just that. Have you tested it with these events?

Reply:
that is true, i'll change it